app/helpers/application_helper.rb

```diff
@@ -2,4 +2,12 @@ module ApplicationHelper
   def last_updated_date
     File.mtime(Rails.root.join('REVISION')).to_date rescue Date.today
   end
+
+  def ajax_enabled_for?(name)
+    %w(
+      energy-grants-calculator
+      calculate-employee-redundancy-pay
+      register-a-death
+    ).exclude?(name)
```

Comment:
This is actually how I did it at first. But then I had a chat with @pmanrubia and he said it probably shouldn't be part of a flow.